javascript/modules/client/src/client.ts

```diff
@@ -630,6 +692,85 @@ export class Client extends ClientCore implements IClient {
     return Promise.resolve({ deposits, withdrawals });
   }
 
+  private _buildGrantAction(
+    params: IPermissionParams,
+  ): DaoAction {
+    const signer = this.web3.getSigner();
+    if (!signer) {
+      throw new Error("A signer is needed");
+    }
+    let { where, who } = params;
+    if (!isAddress(where)) {
+      throw new Error("Invalid address");
+    }
+    if (!isAddress(who)) {
+      throw new Error("Invalid address");
+    }
+    return {
+      to: where,
```

Comment:
From what I understood reading the contracts `where` is the dao address where the permissions are granted, `who` is the address of whom the permissions are granted to.

So here the `where` param will be the dao address. Is this incorrect?

Reply:
It doesn't have to be the DAO. The DAO's permissions database could contain rules affecting 100% external contracts. For example, a plugin could request a rule so that only helper `0x1234` was able to `MINT` on helper `0x2345`. None of which would be the DAO neither the Plugin

Reply:
Oh, ok, that makes sense, I'll add the dao address as a parameter then